src/gui/properties/trackerlistwidget.cpp

```diff
@@ -356,6 +356,46 @@ void TrackerListWidget::loadTrackers()
 
     loadStickyItems(torrent);
 
+    const auto setAlignment = [](QTreeWidgetItem *item)
+    {
+        for (TrackerListColumn col : {COL_TIER, COL_PEERS, COL_SEEDS, COL_LEECHES, COL_TIMES_DOWNLOADED})
```

Comment:
Wrapping this code into lambda is redundant. Could you use it directly?

Reply:
Yes, it would be useful for next commit, but ok.